### ISSUE TITLE
ALTS causing issues with ordering

### ISSUE THREAD

**Issue by brainstorm** [sigven/pcgr#5]
Hello PCGR

The approach used to [reorder the VCF files with egrep](https://github.com/sigven/pcgr/blob/master/src/pcgr/pcgr_check_input.py#L178) affects VCFs generated with alternative chrom names such as:

```
(...)
GL000192.1      123453  .       C       T       169.0   PASS
GL000220.1      42344   .       T       C       .           PASS 
```

Then the PCGR process fails and no reports are generated since `tabix` fails ealier with `[E::hts_idx_push] chromosome blocks not continuous`:

```
2017-05-22 15:41:56 - pcgr-validate-input - INFO - STEP 0: Validate input data
INFO:pcgr-validate-input:STEP 0: Validate input data
2017-05-22 15:41:57 - pcgr-check-input - INFO - Validating VCF file with EBIvariation/vcf-validator
2017-05-22 15:41:58 - pcgr-check-input - INFO - VCF file /workdir/input_vcf/cov50-ensemble-annotated-decomposed.vcf.gz is valid according to v4.2 specification
2017-05-22 15:41:58 - pcgr-check-input - INFO - Checking if existing INFO tags of query VCF file coincide with PCGR INFO tags
2017-05-22 15:41:58 - pcgr-check-input - INFO - No query VCF INFO tags coincide with PCGR INFO tags
[E::hts_idx_push] chromosome blocks not continuous
tbx_index_build failed: /workdir/output/cov50-ensemble-annotated-decomposed.pcgr_ready.vcf.gz
2017-05-22 15:41:58 - pcgr-validate-input - INFO - Finished
INFO:pcgr-validate-input:Finished
2017-05-22 15:41:58 - pcgr-vep - INFO - STEP 1: Basic variant annotation with Variant Effect Predictor (v85, GENCODE, GRCh37)
INFO:pcgr-vep:STEP 1: Basic variant annotation with Variant Effect Predictor (v85, GENCODE, GRCh37)
2017-05-22 15:41:59 - Read existing cache info
2017-05-22 15:41:59 - Starting...
2017-05-22 15:41:59 - Detected format of input file as vcf
WARNING: Chromosome GL000208.1 not found in cache on line 315
WARNING: Chromosome GL000217.1 not found in cache on line 318
WARNING: Chromosome GL000203.1 not found in cache on line 320
WARNING: Chromosome GL000214.1 not found in cache on line 323
WARNING: Chromosome GL000217.1 not found in cache on line 325
WARNING: Chromosome GL000217.1 not found in cache on line 326
WARNING: Chromosome GL000217.1 not found in cache on line 329
WARNING: Chromosome GL000217.1 not found in cache on line 330
WARNING: Chromosome GL000217.1 not found in cache on line 336
WARNING: Chromosome GL000217.1 not found in cache on line 339
WARNING: Chromosome GL000217.1 not found in cache on line 341
WARNING: Chromosome GL000217.1 not found in cache on line 342
WARNING: Chromosome GL000217.1 not found in cache on line 344
2017-05-22 15:42:01 - Read 2232 variants into buffer
2017-05-22 15:42:01 - Calculating consequences
2017-05-22 15:42:41 - Writing output
2017-05-22 15:42:41 - Processed 2232 total variants (53 vars/sec, 53 vars/sec total)
2017-05-22 15:42:41 - Wrote stats summary to /workdir/output/cov50-ensemble-annotated-decomposed.pcgr_ready.pcgr_vep.vcf.tmp_summary.html
2017-05-22 15:42:41 - See /workdir/output/cov50-ensemble-annotated-decomposed.pcgr_ready.pcgr_vep.vcf.tmp_warnings.txt for details of 13 warnings
2017-05-22 15:42:41 - Finished!
[E::hts_idx_push] chromosome blocks not continuous
tbx_index_build failed: /workdir/output/cov50-ensemble-annotated-decomposed.pcgr_ready.pcgr_vep.vcf.gz
```

thanks @ohofmann and @chapmanb for pointing this out.

**Comment by sigven** [comment]
hi @brainstorm (and @ohofmann and @chapmanb),
Thanks a lot for reporting this. Obviously this is something that should be handled. Would you be able to share your input file in some way?

**Comment by brainstorm** [comment]
@sigven I tried to trim down the input file to give you a testcase but the error vanished. Would you mind if I share it with you in private?

**Comment by sigven** [comment]
@brainstorm Sure, that was my intention too. Email is sigven AT ifi DOT uio DOT no.

**Comment by sigven** [comment]
Fixed in 0.4.0
